gitgoing/gitgoing.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+NEURON_COMPONENTS = ('soma', 'axon', 'dendrite', 'synapse')
+
+def mean(x):
```

Comment:
Or even just call it `mean_plus_one`, might actually help clarify the whole plus 1 business